irida_import/config.ini.sample

```diff
@@ -21,3 +21,15 @@ token_endpoint_suffix: /api/oauth/token
 client_id: auth_code_client
 irida_url: http://localhost:8080
 initial_endpoint_suffix: /projects
+
+# Irida file storage mechanism (azure, aws, local)
+irida_storage_type: local
+
+
+# Azure configuration
+azure_container_name: AZURE_CONTAINER_NAME
```

Comment:
Since the sample config uses local by default can you comment out the azure and aws options.

Reply:
Updated in 50e1f7291bd06d1da1f8fe3f432dacba6314d133